java/java.lsp.server/build.xml

```diff
@@ -67,7 +67,7 @@
                 <isset property="metabuild.hash" />
             </not>
         </condition>
-        <property name="vsix.version" value="0.1.0"/>
+        <property name="vsix.version" value="12.3.1"/>
```

Comment:
Right, this was used for testing only, as version 0.1.0 is hard to test, because of the higher version being available on the marketplace. Thanks for notice, I'm reverting this.